build.gradle

```diff
@@ -42,7 +43,7 @@ gradlePlugin {
         commonCustomUserData {
             id = 'com.gradle.common-custom-user-data-gradle-plugin'
             displayName = 'Gradle Enterprise Common Custom User Data Gradle Plugin'
-            description = 'A Gradle plugin to capture common custom user data used for Gradle Build Scans in Gradle Enterprise'
+            description = releaseNotes().get()
```

Comment:
You are reading the release notes twice. I suggest you pull this into a variable called releaseNotes that you declare above the `repositories` block.